react/src/reactTextMask.js

```diff
@@ -19,14 +19,17 @@ export const MaskedInput = React.createClass({
   },
 
   componentDidMount() {
-    const {props, props: {value}} = this
+    this.textMaskInputElement = createTextMaskInputElement()
+    this.update()
+  },
 
-    this.textMaskInputElement = createTextMaskInputElement({inputElement: this.inputElement, ...props})
-    this.textMaskInputElement.update(value)
+  update(value) {
+    const textMaskConfig = Object.assign({inputElement: this.inputElement}, this.props)
```

Comment:
seems we do have [`Object assign pollyfilly`](https://github.com/text-mask/text-mask/blob/master/package.json#L81)